engine/security/src/main/scala/pl/touk/nussknacker/ui/security/api/AuthenticationConfiguration.scala

```diff
@@ -62,6 +62,15 @@ object AuthenticationConfiguration {
                         categories: List[String] = List.empty,
                         permissions: List[Permission] = List.empty,
                         globalPermissions: List[GlobalPermission] = List.empty)
+
+  private def parseUsers(usersLocation: String) = {
```

Comment:
Good idea. Can we either:
- extract it to utils and merge somehow with logic in CustomFicusInstances.urlValueReader 
- use sth like ResourceLoader (I know that solution below is somewhat nicer, as it doesn't require us to write "file://"